### ISSUE TITLE
Forbid use of EXHIBIT keyword

### ISSUE THREAD

**Issue by rooksdo** [TypeCobolTeam/TypeCobol#1946]
### Description
OS/VS COBOL keyword `EXHIBIT` is not supported by IBM compiler. An error should be detected for this instruction specifically in EI environment.

Sample
```cob
       IDENTIFICATION DIVISION.
       PROGRAM-ID. foo.
       DATA DIVISION.
       WORKING-STORAGE SECTION.
       01 WS-FOO PIC X(42).
       01 WS-BAR PIC X(42).
       PROCEDURE DIVISION.
      * Non-Compliant
           EXHIBIT NAMED WS-FOO WS-BAR.
       END PROGRAM foo.
```

**Expected result**
"E-I compilation doesn't allow use of EXHIBIT instruction."

**Comment by fm-117** [comment]
Current diagnostic for this sample is: `Line 9[20,24] <27, Error, Syntax> - Syntax error : no viable alternative at input 'EXHIBIT ... NAMED'`

**Comment by fm-117** [comment]
Closed with PR #1959, a new customized messsage is used for unsupported keywords.
